runtime/runtime.go

```diff
@@ -275,51 +279,19 @@ func (r *interpreterRuntime) ExecuteTransaction(
 	}
 
 	_, err = r.interpret(
+		location,
 		runtimeInterface,
 		runtimeStorage,
 		checker,
 		functions,
 		nil,
-		func(inter *interpreter.Interpreter) (interpreter.Value, error) {
```

Comment:
Just refactored that into its own function, i.e. https://github.com/onflow/cadence/pull/77/files#diff-6681b213d55a5c65c5d90b211107fd52R306 is identical